modules/react/modal/lib/ModalHeading.tsx

```diff
@@ -19,6 +19,10 @@ const ResponsiveModalHeading = styled(Popup.Heading)<ModalHeadingProps & StyledT
   return {
     [canvasTheme.breakpoints.down('s')]: {
       marginBottom: space.zero,
+      paddingTop: space.xxs,
```

Comment:
```suggestion
      padding: `${space.xxs} ${space.xxs} ${space.xxxs} ${space.xxs}`,
```
this will add padding top xxs right xxs bottom xxxs left xxs